charts/capsule-proxy/README.MD

```diff
@@ -0,0 +1,132 @@
+
+    * PodNodeSelector
+    * LimitRanger
+    * ResourceQuota
+    * MutatingAdmissionWebhook
+    * ValidatingAdmissionWebhook
+
+* A [`kubeconfig`](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) file accessing the Kubernetes cluster with cluster admin permissions.
+
+## Quick Start
+
+The Capsule-proxy Chart can be used to instantly deploy the Capsule-proxy on your Kubernetes cluster.
+
+1. Add this repository:
+
+        $ helm repo add clastix https://clastix.github.io/charts
+
+2. Install the Chart:
+
+        $ helm install capsule-proxy clastix/capsule-proxy -n capsule-system
+
+3. Show the status:
+
+        $ helm status capsule-proxy -n capsule-system
+
+4. Upgrade the Chart
+
+        $ helm upgrade capsule-proxy clastix/capsule-proxy -n capsule-system
+
+5. Uninstall the Chart
+   
+        $ helm uninstall capsule-proxy -n capsule-system
+
+## Customize the installation
+
+There are two methods for specifying overrides of values during chart installation: `--values` and `--set`.
+
+The `--values` option is the preferred method because it allows you to keep your overrides in a YAML file, rather than specifying them all on the command line. Create a copy of the YAML file `values.yaml` and add your overrides to it.
+
+Specify your overrides file when you install the chart:
+
+        $ helm install capsule-proxy clastix/capsule-proxy --values myvalues.yaml -n capsule-system  
+
+The values in your overrides file `myvalues.yaml` will override their counterparts in the chart’s values.yaml file. Any values in `values.yaml` that weren’t overridden will keep their defaults.
+
+If you only need to make minor customizations, you can specify them on the command line by using the `--set` option. For example:
+
+        $ helm install capsule-proxy clastix/capsule-proxy --values myvalues.yaml -n capsule-system
+
+Here the values you can override:
+
+Parameter | Description | Default
+--- | --- | ---
+`image.repository` | Set the image repository of the capsule-proxy. | `quay.io/clastix/capsule-proxy`
+`image.pullPolicy` | Set the image pull policy. | `IfNotPresent`
+`image.tag` | Overrides the image tag whose default is the chart. `appVersion` | `null`
+`options.logLevel` | Set the log verbosity of the capsule-proxy with a value from 1 to 10.| `5`
+`options.k8sControlPlaneUrl` | Set the URL of kubernetes control plane. | `https://kubernetes.default.svc`
+`options.capsuleUserGroup` | Override the Capsule user group | `capsule.clastix.io`
+`options.oidcUsernameClaim` | Override the OIDC field name used to identify the user | `preferred_username`
+`options.enableSSL` | Specify if capsule-proxy will use SSL | `false`
+`options.SSLDirectory` | Set the directory, where SSL certificate and keyfile will be located | `/opt/certs`
```

Comment:
@MaxFedotov default value for certificates should be consistent with default in command options

https://github.com/clastix/capsule-proxy/blob/1419f46292dae7dd4e61cec0af7ddcab98565972/main.go#L41-L42

Should we use two options here:

`options.SSLcertPath`
`options.SSLkeyPath`

instead of:

`options.SSLDirectory`
`options.SSLCertFileName`
`options.SSLKeyFileName`

What do you think?

Reply:
Thanks, now values are consistent.

We need `options.SSLDirectory` in order to configure `volumeMounts` for deployment:
```yaml
          {{- if .Values.options.enableSSL }}
          volumeMounts:
          - mountPath: {{ .Values.options.SSLDirectory }}
            name: certs
          {{- end }}
```

Reply:
LGTM